media/common/src/main/java/io/helidon/media/common/MediaSupport.java

```diff
@@ -61,27 +90,62 @@ public static MediaSupport createWithDefaults() {
         return builder().registerDefaults().build();
     }
 
+    /**
+     * Create a new instance with the default readers and writers registered on
+     * the contexts.
+     * @param config a {@link Config} that will be passed to {@link
+     * Builder#registerDefaults(Config)}
+     * @return MediaSupport
+     */
+    public static MediaSupport createWithDefaults(Config config) {
+        return builder(config).registerDefaults().build();
+    }
+
     /**
      * Create a new {@link Builder} instance.
-     * @return Builder
+     * @return a new {@link Builder}
      */
     public static Builder builder() {
         return new Builder();
     }
 
+    /**
+     * Create a new {@link Builder} instance.
+     * @param config a {@link Config}
+     * @return a new {@link Builder}
+     */
+    public static Builder builder(Config config) {
```

Comment:
In other places where we use config, we usually create the following:
- static factory method `builder()` - already in this class
- method `Builder.config(Config)` - already in builder
- static factory method `create(Config)` 

this method is not that useful 
- if you want to create `MediaSupport` from config only (if that is even feasible), add the `create(Config)` method
- if you want to update the builder from config and then do more with the builder, just use `builder().config(Config)`

This shortcut does not seem useful.